tests/test_plugin.py

```diff
@@ -3,12 +3,13 @@
 from freezegun import freeze_time
 from mock import call, create_autospec
 import pytest
-
+import sys
+import os
 from pytest_testrail import plugin
 from pytest_testrail.plugin import PyTestRailPlugin, TESTRAIL_TEST_STATUS
 from pytest_testrail.testrail_api import APIClient
 
-
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
```

Comment:
could you enlighten me what this line does?

Reply:
yeah that's unnecessary code.
Thanks Peter